daemon/ZilliqaDaemon.cpp

```diff
@@ -15,95 +15,158 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#include <dirent.h>
-#include <errno.h>
-#include <signal.h>
-#include <stdlib.h>
-#include <sys/stat.h>
-#include <sys/types.h>
-#include <sys/wait.h>
-#include <unistd.h>
-#include <algorithm>
-#include <array>
-#include <cstdio>
-#include <fstream>
-#include <iostream>
-#include <memory>
-#include <stdexcept>
-#include <string>
-#include <unordered_map>
-#include <vector>
+#include "ZilliqaDaemon.h"
 
 using namespace std;
-
-const vector<string> programName = {"zilliqa", "lzilliqa", "latezilliqa"};
-
-const string restart_zilliqa = "python tests/Zilliqa/daemon_restart_local.py";
-
-unordered_map<int, string> PrivKey;
-unordered_map<int, string> PubKey;
-unordered_map<int, string> Port;
-unordered_map<int, string> Path;
-unordered_map<int, bool> isDS;
+namespace po = boost::program_options;
+
+#define SUCCESS 0
+#define ERROR_IN_COMMAND_LINE -1
+
+const vector<string> programName = {"zilliqa"};
+const string SYNCTYPE_OPT = "--synctype";
+const char* synctype_descr =
+    "0(default) for no, 1 for new, 2 for normal, 3 for ds, 4 for lookup, 5 "
+    "for node recovery, 6 for new lookup , 7 for ds guard node sync and 8 "
+    "for offline validation of DB";
+
+const string scripts_path = "/run/zilliqa/";
```

Comment:
Can we try to parameterize these path constants? Otherwise, users are forced to follow the paths you hardcoded.

Reply:
Ok, will put to `constants.xml` also